api-reference/10 UI Components/dxValidator/8 Validation Rules/PatternRule/PatternRule.md

```diff
@@ -15,5 +15,7 @@ To specify the regular expression that the validated field must match, set the r
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/"
 }
 
```

Comment:
Can you update line 9 (above) as follows?
`A validation rule that demands that the validated field match a specified pattern.`

Please change "demands" to "requires".